web/packages/teleport/src/Discover/SelectResource/resources/resourcesE.tsx

```diff
@@ -59,4 +59,21 @@ export const SAML_APPLICATIONS: SelectResourceSpec[] = [
     icon: 'googlecloud',
     event: DiscoverEventResource.SamlApplication,
   },
+  {
+    id: DiscoverGuideId.ApplicationSamlMicrosoftEntraId,
+    name: 'Azure Console Access',
```

Comment:
Pending confirmation, this integration might enable Azure CLI access too. If it does, then I think generic name "Microsoft Entra ID" would be better and I will follow up here.

Reply:
The integration works for Azure CLI too. I renamed the tile as "Microsoft Entra External ID" in https://github.com/gravitational/teleport/pull/53826/commits/31447159e90a6af903aa8475872e67d95c165105. 
This aligns with the official Microsoft product name https://learn.microsoft.com/en-us/entra/external-id/external-identities-overview